### ISSUE TITLE
Any plans on maknig a release / publishing the package on PyPI

### ISSUE THREAD

**Issue by isidentical** [intake/gdrivefs#21]
Apparently, the name of the package on PyPI is already in use, so maybe it can be published under something else like `gdrive-spec`?

**Comment by martindurant** [comment]
I agree generally that this should be done, but we don't have documentation or testing yet in this repo, so it's state is "very alpha". I'm not sure anyone will have the time to fix that.

**Comment by isidentical** [comment]
Is there any need for extra documentation though? fsspec already covers the public API

**Comment by martindurant** [comment]
It would be nice to have some minimal documentation, particularly about how to auth. Maybe the docstring is enough.
